lib/cartopy/mpl/ticker.py

```diff
@@ -468,14 +468,11 @@ class LongitudeLocator(MaxNLocator):
         Allow the locator to stop on on minutes and seconds (False by default)
     """
 
-    default_params = MaxNLocator.default_params.copy()
-    default_params.update(nbins=8, dms=False)
-
     def set_params(self, **kwargs):
         """Set parameters within this locator."""
-        MaxNLocator.set_params(self, **kwargs)
-        if 'dms' in kwargs:
-            self._dms = kwargs['dms']
+        nbins = kwargs.pop('nbins', 8)
```

Comment:
This won't work, because `__init__` will use `nbins` from `default_params` always.

Reply:
OK, that is a bonkers way to set default params, but I think the new change works.  Not quite sure if the documentation gets inherited the right way...